api/websocket/server/server.go

```diff
@@ -237,7 +240,7 @@ func (ws *WsServer) Restart() {
 		time.Sleep(time.Second)
 		ws.Stop()
 		time.Sleep(time.Second)
-		go ws.Start()
+		go ws.Start(nil)
```

Comment:
nil channel will never fire any event, so wss ready will never be fired

Reply:
this function were unused, should we delete it?

Reply:
I think we can delete it